web-client/src/presenter/sequences/navigateToEditOrderSequence.ts

```diff
@@ -29,16 +30,22 @@ export const navigateToEditOrderSequence = [
   setFormFromDraftStateAction,
   setDocumentToEditAction,
   removeSignatureAction,
-  isStatusReportOrderAction,
+  checkDocumentTypeAction,
   {
-    isNotStatusReportOrder: [
-      getDocumentEditUrlAsPathAction,
-      navigateToPathAction,
-    ],
-    isStatusReportOrder: [
-      setEditStatusReportOrderFormAction,
-      navigateToPathAction,
-      statusReportOrderPdfPreviewSequence,
+    documentTypeMiscellaneous: [navigateToPathAction],
```

Comment:
Unfortunately, something isn't working quite right here. If I try to edit a status report order, I am navigated to the upload screen:

https://github.com/user-attachments/assets/ad61d684-af8f-421b-ad29-14ec976e03b0

Reply:
This is such a crucial catch! I appreciate your vigilance.

The root cause of the bug you recorded was that the logic in `isMiscellaneousDocketEntry` contained an insufficient check to determine whether the docket entry at hand is a status report order: previously, it was checking to make sure that the `docketEntry.draftOrderState.orderType` property was keyed to a valid value.

However, as you demonstrated in the video above, this does not cover the case in which a status report order is created without the user having selected an order type.

To address this, I've broken the logic that determines if a given docket entry is a draft status report order into its own function (see below). As best I can tell, `draftOrderState.statusReportFilingDate` and `draftOrderState.statusReportIndex` will always be set for any draft status report order no matter what the circumstances. 

```typescript
export const isDraftStatusReportOrder = (docketEntry): boolean => {
  // At an absolute minimum, a draft status report order will have
  // docketEntry.draftOrderState.statusReportFilingDate and
  // docketEntry.draftOrderState.statusReportIndex
  return !!(
    docketEntry &&
    typeof docketEntry === 'object' &&
    docketEntry.draftOrderState &&
    typeof docketEntry.draftOrderState === 'object' &&
    docketEntry.draftOrderState.statusReportFilingDate &&
    docketEntry.draftOrderState.statusReportIndex
  );
};
```

`isMiscellaneousDocketEntry` now uses this function.